Rakefile

```diff
@@ -25,6 +26,7 @@ require 'benchmark'
 require 'chronic_duration'
 
 dry_run = (ENV["DRY_RUN"].nil? || ENV["DRY_RUN"].empty?) ? false : true
+official_pkg = ENV["OFFICIAL_FLAPJACK_PACKAGE"] == 'true'
```

Comment:
Can this be handled the same was as DRY_RUN? That is, if the env var is set and is non-empty, then it's taken to be true?